extensions/debug-auto-launch/src/extension.ts

```diff
@@ -5,41 +5,61 @@
 
 import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
+import { createServer, Server } from 'net';
 
 const localize = nls.loadMessageBundle();
-const ON_TEXT = localize('status.text.auto.attach.on', "Auto Attach: On");
-const OFF_TEXT = localize('status.text.auto.attach.off', "Auto Attach: Off");
+const ON_TEXT = localize('status.text.auto.attach.on', 'Auto Attach: On');
+const OFF_TEXT = localize('status.text.auto.attach.off', 'Auto Attach: Off');
```

Comment:
with single ticks for the default localisation strings you are in the minority 😁 (1500 vs. 5000).